src/main/java/uk/gov/justice/hmpps/prison/service/MovementUpdateService.java

```diff
@@ -46,7 +46,7 @@ public MovementUpdateService(
     }
 
     @Transactional
-    @VerifyBookingAccess
+    @VerifyBookingAccess(overrideRoles = {"ROLE_SYSTEM_USER", "ROLE_UNILINK"})
```

Comment:
this is adding a new role, since I've left `SYSTEM_USER` in there it will work for any existing users calling the booking endpoint